### PR TITLE
fix(Upload): unnecessary spacing when not providing `title` and `text`

### PR DESCRIPTION
**Pull request by langz**

### CODE REVIEW

File: packages/dnb-eufemia/src/components/upload/UploadInfo.tsx [hunk 1]
```diff
@@ -57,19 +58,18 @@ const UploadInfo = () => {
     displayFilesAmountLimitItem
 
   return (
-    <>
-      {title && <Lead space="0">{title}</Lead>}
+    <Flex.Stack gap="small">
+      {(title || text) && (
+        <Flex.Stack gap="xx-small">
```
Comment (langz) [inline packages/dnb-eufemia/src/components/upload/UploadInfo.tsx:63]: You could say this Flex component is not needed if there's only a `title` or a `text` provided, and not both.
But not sure if it's disturbing to have the Flex.Stack wrapper in that scenario.

File: packages/dnb-eufemia/src/components/upload/__tests__/__image_snapshots__/upload-for-ui-have-to-match-file-max-size-based-on-file-type-table.snap.png
Comment (langz) [inline packages/dnb-eufemia/src/components/upload/__tests__/__image_snapshots__/upload-for-ui-have-to-match-file-max-size-based-on-file-type-table.snap.png:1]: I have to fix this.

Comment (langz) [reply packages/dnb-eufemia/src/components/upload/__tests__/__image_snapshots__/upload-for-ui-have-to-match-file-max-size-based-on-file-type-table.snap.png:1]: Fixed

**Review by tujoworker — APPROVED** [review]

**Pull request merged**